pyarlo/base_station.py

```diff
@@ -19,11 +24,144 @@ def __init__(self, name, attrs, arlo_session):
         self.name = name
         self._attrs = attrs
         self._session = arlo_session
+        self._session_token = session_token
+        self.__sseclient = None
+        self.__subscribed = False
+        self.__events = []
 
     def __repr__(self):
         """Representation string of object."""
         return "<{0}: {1}>".format(self.__class__.__name__, self.name)
 
+    def thread_function(self):
+        """Thread function."""
+        url = SUBSCRIBE_ENDPOINT + "?token=" + self._session_token
+        data = self._session.query(url, method='GET', raw=True, stream=True)
+        self.__sseclient = sseclient.SSEClient(data)
+        if self.__sseclient:
+            self.__subscribed = True
+        for event in (self.__sseclient).events():
+            if not self.__subscribed:
+                break
+            data = json.loads(event.data)
+            if 'status' in data:
+                if data['status'] == "connected":
+                    _LOGGER.debug("Successfully subscribed this base station")
+            elif 'action' in data:
```

Comment:
Status and output are not available at the same time, in the event stream output.

'status' is available only on the connection response that we get first.  After that, all the notifications have only the 'action' in the event stream.

That's why I added the 'status' and 'action' as if-else statement